src/internet_identity/src/openid.rs

```diff
@@ -140,6 +140,29 @@ impl OpenIdCredential {
             }
         })
     }
+
+    /// Find current config for stored credential
+    pub fn config_issuer(&self) -> Option<String> {
```

Comment:
Could we add some tests for this? Or too much effort?

Maybe if we move the function inside "find" as a pure function and test that?